### ISSUE TITLE
Don't require msilib on other platforms than Windows

### ISSUE THREAD

**Issue by xmikos** [xoviat/subzero#53]
Hello,
I want to use *subzero* mainly for its PyInstaller setuptools integration (so only `build_exe` command is relevant for me), but it seems not to be possible right now because of `msilib` import:

```
In [1]: from subzero import setup
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-8f229b3286a7> in <module>()
----> 1 from subzero import setup

/home/mikos/.local/lib/python3.6/site-packages/subzero/__init__.py in <module>()
     10 
     11 from subzero.dist import build_exe, Executable
---> 12 from subzero.windist import bdist_msi
     13 
     14 # This file was originally taken from cx_Freeze by Anthony Tuininga, and is licensed under the  PSF license.

/home/mikos/.local/lib/python3.6/site-packages/subzero/windist.py in <module>()
      1 # This file was originally taken from cx_Freeze by Anthony Tuininga, and is licensed under the  PSF license.
      2 
----> 3 import distutils.command.bdist_msi
      4 import distutils.errors
      5 import distutils.util

/usr/lib/python3.6/distutils/command/bdist_msi.py in <module>()
     15 from distutils.util import get_platform
     16 from distutils import log
---> 17 import msilib
     18 from msilib import schema, sequence, text
     19 from msilib import Directory, Feature, Dialog, add_data

ModuleNotFoundError: No module named 'msilib'
```

Can you please make this dependency optional (only provide `bdist_msi` on Windows, but make `build_exe` available on other platforms too)?

**Comment by xoviat** [comment]
Can you try installing the `linux_support` branch?

```
pip install git+https://github.com/xoviat/subzero.git@linux_support
```

**Comment by xmikos** [comment]
Great, this seems to work, I don't get exception anymore!

Btw. if I don't want to do `from subzero import setup`, but only `from subzero import build_exe` and add build_exe to `cmdclass` manually, is this supported? I have just tried it and it seems to work too (it didn't work with cx_Freeze, so this seems to be another great improvement in subzero ;-)).

**Comment by xoviat** [comment]
> Btw. if I don't want to do from subzero import setup, but only from subzero import build_exe and add build_exe to cmdclass manually, is this supported? I have just tried it and it seems to work too (it didn't work with cx_Freeze, so this seems to be another great improvement in subzero ;-)).

There is some preprocessing done in the subzero setup function, for two reasons:
- to automatically install project dependencies (install_requires) before building
- to allow the use of the executable class (see README)

**Comment by xoviat** [comment]
Closed in 0ccf0c956dba2502acd99ce1b519ed7e4dc072ff.
